app/modules/home.py

```diff
@@ -1,12 +1,17 @@
 import webapp2
 import logging
 import os
+import geocoder
```

Comment:
`geocoder` isn't being used - let's remove it from here and `requirements.txt`

Reply:
I wanted the app to get the user coordinates as default for wiki api, I will finish it after the tasks.
any way I will remove it for now